tests/tools/saunafsXX.sh

```diff
@@ -93,6 +77,17 @@ saunafsXX_master_daemon() {
 	return ${PIPESTATUS[0]}
 }
 
+saunafsXX_shadow_daemon_n() {
+	local id=$1
+	shift
+	if is_windows_system; then
```

Comment:
> Also, I would like to point out that you added the Windows check here for the shadow, but not for the master or chunkserver, do you have any reasoning behind this?

Because I just copied the function over and changed the variable. Yeah it could be done better, but there might be reasons we may want to keep separate and there's really no benefit in over-abstracting right now, but if we start repeating these patterns we might want to create something.

As for getting it working on Windows, I'll leave that mess to you :)